charts/alertmanager/values.yaml

```
@@ -251,3 +251,9 @@ extraVolumes: []
 extraEnv: []
   # - name: FOO
   #   value: BAR
+
+testFramework:
+  enabled: true
```

Should we enable that by deafult?
I would let it disabled.

```suggestion
  enabled: false
```